### ISSUE TITLE
Menubar appearance under Unity

### ISSUE THREAD

**Issue by aerodynamic-sauce-pan** [EliverLara/Sweet#45]
Hello,

First of all let me tell you I'm totally found of your work with this theme. Love it !
But I have encountered a strange behavior under Ubuntu 19.04 Disco **Unity** (works fine under Gnome3) and I'm not sure if it's a genuine bug or if something else is interfering.

![issue_sweet-dark_menu-bar](https://user-images.githubusercontent.com/42494948/63091299-39db1180-bf5e-11e9-9e93-817003a33fb7.png)

In the picture you can see nautilus sessions (from top to bottom) : opened with root privileges and in background, opened without root privileges and in background, opened without root privileges and in foreground. 

Basically in root mode the menu bar is behaving normally, as in your screenshots (whatever the app) but without, it's replaced with this light gray colored gradient Unity-style-shapped bar. Same problem seems to apply to the desktop menu bar.

I have tried to change some variables background colors in the gtk and gtk-dark css, unsuccessfully... Any idea why ?

PS : I know Unity in no longer supported but I still find it more practical and smooth than Gnome3 which is why I prefer to stick to it for now.

**Comment by EliverLara** [comment]
Could you add [this](https://github.com/EliverLara/Sweet/files/3510555/unity.zip) folder to the theme, that should fix the window buttons style.

About the color of the bar, you could try changing the colors in the `gtk-3.0/apps/_unity.scss` file and then recompiling the theme

**Comment by aerodynamic-sauce-pan** [comment]
The folder did do the trick for the window buttons as expected.

Regarding the bar, I expect I should clone your master branch (given I aim to use Sweet-dark), modify `gtk-3.0/apps/_unity.scss`, recompile then add the Unity folder back. But I'm a total stranger to node js and seem to have broken my npm with the "Error: Cannot find module 'semver'" I cannot resolve... I'll let you know, if I succeed to fix that, if simply changing the colors of `gtk-3.0/apps/_unity.scss` is enough to do the trick.

But even if changing the color would make things better, the 'Unity' bar still is on top of the application's own bar (when it has one, say Nautlius). Is there anything that can be done about that ?
